lib/steep/server/worker_process.rb

```diff
@@ -40,9 +40,13 @@ def self.spawn_worker(type, name:, steepfile:, options: [], delay_shutdown: fals
         new(reader: reader, writer: writer, stderr: stderr, wait_thread: thread, name: name)
       end
 
-      def self.spawn_typecheck_workers(steepfile:, count: [Etc.nprocessors - 1, 1].max, delay_shutdown: false)
+      def self.spawn_typecheck_workers(steepfile:, args:, count: [Etc.nprocessors - 1, 1].max, delay_shutdown: false)
```

Comment:
The command-line arguments forwarding from the master process to worker processes are done with command-line arguments.